make.cmd

```diff
@@ -18,6 +18,16 @@ DEL /F /A /Q y.output
 
 golex -o parser/scanner.go parser/scanner.l
 
+@echo [Ast-Parser]
+go get github.com/qiuyesuifeng/goyacc
+go get github.com/qiuyesuifeng/golex
+type nul >>temp.XXXXXX
+goyacc -o nul -xegen "temp.XXXXXX" parser/parser.y
```

Comment:
in Makefile

`goyacc -o /dev/null -xegen $$a parser/parser.y; \`
`goyacc -o /dev/null -xegen $$a ast/parser/parser.y; \`

same to make.cmd